libs/core/src/scss/interaction-state/_active.scss

```diff
@@ -0,0 +1,10 @@
+@use 'layer';
+
+$default-loudness-active: 'l';
```

Comment:
Quick question - any reason to not put this in `_interaction-state.utilities`?

Reply:
No. Only that I created it _before_ the utilities issue was merged 🙂